runtime/queries/apex/locals.scm

```diff
@@ -1,41 +1,51 @@
 ; declarations
-
 (class_declaration) @local.scope
+
 (method_declaration) @local.scope
+
 (constructor_declaration) @local.scope
+
 (enum_declaration) @local.scope
+
 (enhanced_for_statement) @local.scope
 
 ; if/else
-
 (if_statement) @local.scope
+
 (if_statement
-  consequence: (_) @local.scope) ; if body in case there are no braces
+  ; if body in case there are no braces
+  consequence: (_) @local.scope)
+
 (if_statement
-  alternative: (_) @local.scope) ; else body in case there are no braces
+  ; else body in case there are no braces
+  alternative: (_) @local.scope)
 
 ; try/catch
+; covers try+catch, individual try and catch are covered by (block)
+(try_statement) @local.scope
 
-(try_statement) @local.scope ; covers try+catch, individual try and catch are covered by (block)
-(catch_clause) @local.scope ; needed because `Exception` variable
+; needed because `Exception` variable
+(catch_clause) @local.scope
```

Comment:
Something is still not working correctly with inline comments. They should either all move to the line above, or just be left alone (possibly with spacing adjusted and empty comments trimmed, if that is feasible). My preference is for the latter, as that seems easiest and most robust.

Reply:
I'll try. Probably need a new predicate for that

Reply:
I'd be open to improving the query parser to be able to distinguish inline comments.